### DISCUSSION TITLE
Wiki dissapeared

### DISCUSSION THREAD

**Discussion by orchowski** [nvim-neorg/neorg#1268] [General]
i don't know, am i dumb or something? I've just started using neorg few days ago and i'm pretty sure i saw wiki page with modules and configuration but now i have no access to it.

to be more precise: I see only "dependencies" page

**Comment by tamton-aquib** [comment]
![Screenshot_2024-01-07-08-33-18-19_572064f74bd5f9fa804b05334aa4f912](https://github.com/nvim-neorg/neorg/assets/77913442/e6333d98-d101-4f7d-b3ce-69c89aeaded4)

**Comment by vhyrro** [comment]
Fixed now! :)

**Reply by orchowski** [reply]
thank you!
